app/models/ability.rb

```diff
@@ -0,0 +1,19 @@
+class Ability
```

Comment:
Please use CanCanCan for Post authorization in Post Controller. Please see the example below.
```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

Reply:
Fixed